src/KafkaFlow/Event.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace KafkaFlow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    internal class Event<TArg> : IEvent<TArg>
```

Comment:
I believe this should be public and also inside the `KafkaFlow.Abstractions`

Reply:
What is the rational of having this implementation in KafkaFlow.Abstractions?